cache/cache.go

```diff
@@ -204,7 +217,23 @@ func (r *Repository) loadFromRemote() error {
 }
 
 func (r *Repository) makeCacheDir() error {
-	return os.MkdirAll(r.directory, 0755)
+	if err := os.MkdirAll(r.directory, 0755); err != nil {
+		fmt.Errorf("ERROR: creating directory %s: %s", r.directory, err)
```

Comment:
Why do you define that error here but returning the original error?
Shouldn't that be the error you are returning?